src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -1554,6 +1563,55 @@ protected function hasChanges($changes, $attributes = null)
         return false;
     }
 
+    /**
+     * Determine if the attribute has transitioned to a given value.
+     *
+     * @param  string  $attribute
+     * @param  string|array  $to
+     * @param  string|array  $from
+     * @return bool
+     */
+    public function transitionedTo($attribute, $to, $from = null)
+    {
+        if (! array_key_exists($attribute, $this->old) ||
+            ! array_key_exists($attribute, $this->changes)) {
+            return false;
+        }
+
+        $oldValue = is_null($this->old[$attribute]) ? null : (string) $this->old[$attribute];
+        $newValue = is_null($this->changes[$attribute]) ? null : (string) $this->changes[$attribute];
+
+        if (func_num_args() == 3 && ! in_array($oldValue, is_array($from) ? $from : [$from])) {
```

Comment:
Can we use `Arr:wrap()` here and similar places?
```suggestion
        if (func_num_args() == 3 && ! in_array($oldValue, Arr::wrap($from))) {
```

Reply:
Actually breaks the test... slightly different behavior around null.